src/components/modes/teacher/TeacherView.js

```diff
@@ -121,24 +147,27 @@ export class TeacherView extends Component {
     const {
       students,
       comments,
+      teacherComments,
       t,
       dispatchOpenFeedbackView,
       dispatchSetSelectedStudent,
       dispatchGetAppInstanceResources,
       settings: { helpRequestsOnly = DEFAULT_HELP_REQUESTS_ONLY_SETTING },
     } = this.props;
+    const orphans = getOrphans([...comments, ...teacherComments]);
     // if there are no resources, show an empty table
-    if (!comments.length) {
+    if (comments.length - orphans.length <= 0) {
       return (
         <TableRow>
-          <TableCell colSpan={4}>No App Instance Resources</TableCell>
+          <TableCell colSpan={4}>{t('No Interactions')}</TableCell>
         </TableRow>
       );
     }
     // map each app instance resource to a row in the table
     return students.map(({ id, name }) => {
       const numberOfComments =
-        comments.filter((r) => r.user === id)?.length || 0;
+        comments.filter((r) => r.user === id)?.length -
```

Comment:
Is operator precedence here OK?

Reply:
`-` has a higher precedence than `||` so this means it will do `(comment_length - orpahns_length) || 0` which is indeed not the expected result. It should be `(comment_length || 0) - (orphan_length || 0)`